skimage/segmentation/__init__.py

```diff
@@ -5,6 +5,7 @@
 from ._quickshift import quickshift
 from .boundaries import find_boundaries, mark_boundaries
 from ._clear_border import clear_border
+from .chan_vese import chan_vese
```

Comment:
I see, there is name conflict now between module and function names.
Let's rename file to `_chan_vese.py` and change import to `from ._chan_vese.py import chan_vese`.